vsock_sample/py/README.md

```diff
@@ -0,0 +1,108 @@
+# vsock-sample
+
+A hello-world example for vsock server and client communication.
+
+## Prerequisites
+
+1. Get the `vsock-sample` code. `vsock-sample` is an application
+that can be run either as a server or a client. The client sends the
+“Hello, world!” message to the server. The server receives the message
+and prints it to the standard output.
+
+2. The sample is written and tested in Python 3.7.9. Any Python version
+should work as long as it has AF_VSOCK socket support.
+
+## How to use the enclave as server and the parent instance as client
+
+1. Build the Enclave Image File (EIF) starting from a Dockerfile.server
+in this directory. We chose to start from the python-alpine docker image
+to keep the enclave image as small as possible, but you can also use
+other optimized Docker images to further decrease the final image size.
+
+```
+$ docker build -t vsock-sample-server -f Dockerfile.server .
+```
+
+__Note__: You can use any other port number.
```

Comment:
It might not be clear enough that 5005 in Dockerfile.{server, client} is the configurable port number.
I would move this line above ```docker build``` and reword it like this: "__Note__: You can use any other port number by replacing 5005 in Dockerfile.{client/server} with another value."